playbooks/incident-handling.md

```diff
@@ -0,0 +1,458 @@
+  [Android](https://play.google.com/store/apps/details?id=com.ifountain.opsgenie&hl=en)) is installed and logged
+  in.
+- Double check your preferences for push notifications, text messages, and/or phone calls for alerting in your
+  [OpsGenie user notification settings](https://artsy.app.opsgenie.com/settings/user/notification).
+
+### Enable Slack notifications
+
+Potential incidents are usually raised as threads in [#incidents](https://artsy.slack.com/archives/incidents)🔒.
+Additionally, automated alerts notify the [#alerts](https://artsy.slack.com/archives/alerts)🔒 channel. Both are
+worth monitoring during on-call shifts.
+
+#### On the [mac] desktop
+
+- Open the [#incidents](https://artsy.slack.com/archives/incidents)🔒 or
+  [#alerts](https://artsy.slack.com/archives/alerts)🔒 channel and click the channel name to access channel
+  settings
+- Click the star to add the channel to your starred group
+- In the notifications drop-down, select _Get notifications for all messages_
+
+#### On your mobile device
+
+- Make sure "do not disturb" is turned off on the device itself (i.e., no red is visible on an iPhone's silence
+  switch)
+- In the slack app, click the channel name to access channel settings
+- Click the _Notifications_ option and select _Every new message_
+
+## During your rotation
+
+### Update the #incidents channel's topic with who is on call
+
+- Click the [#incidents](https://artsy.slack.com/archives/incidents)🔒 channel's name
+- Click _Edit_ on the channel's topic
+- Update the topic with the with the new on-call engineers' names (that's you!)
+
+_Tip: use the `/genie whoisoncall` Slack command to find out who is on call at any time._
+
+### 1. Monitor
+
+During an on-call shift, engineers are expected to monitor the [relevant channels](#enable-slack-notifications),
+acknowledge alerts, and respond to incident reporters in a timely and responsible way.
+
+#### Service-Level Objective
+
+We target the following [service-level](https://landing.google.com/sre/sre-book/chapters/service-level-objectives/)
+for incident handling response times:
+
+- **10 minutes** ("full support") during local business hours (9am-5pm)
+- **30 minutes** ("partial support") during local waking hours outside of business hours (7am-7pm on work days, or
+  9am-5pm on non-work days)
+- **no guaranteed response time** ("limited support") beyond those hours
+
+![visualization of our on-call SLO](images/on-call-slo.jpg)
+
+This objective only relates to the time between an incident being reported and the on-call engineers' initial
+acknowledgement (and tracking in Opsgenie). We don't guarantee recovery times, though obviously those are of great
+interest to us.
+
+#### Monitoring automated alerts
+
+The [#alerts](https://artsy.slack.com/archives/alerts) channel receives automated notifications from various
+sources (Datadog, Pingdom, Sentry). It aims to be quiet and restricted to actionable events, but your judgment is
+required to assess their severity.
```

Comment:
**thought:** are you intentionally not linking to our old severity docs here?

Reply:
Sort of intentional. There are 2 types of severity discussed in these docs:
* Here, whether a problem is severe enough to be considered an incident needing on-call support at all. I consider this very important and that's why there are examples below and the repeated, bold rule of thumb about "significant proportion of users, critical business functions, or sensitive data."
* [Those severity docs](https://github.com/artsy/README/blob/43c400d81ff9fee7276c3dd934de26b985da362f/playbooks/support/responding.md#severity-of-incidents), which suggest how to assign priority to something which _is_ deemed an incident. I didn't find those helpful, since they're long and the priority designation ultimately doesn't affect anything.

BUT I wasn't sure about this... if you think some rules of thumb would help, I can bring them back in a collapsed and hopefully more concise section.

Reply:
I also found the severity docs that you linked a bit confusing, I only wanted to mention it in case you missed them. Thanks for explaining

Reply:
If it helps, I think in the past we were exploring the possibility of creating incidents more freely vs. only when the issue is high-severity. If that were the case, the 5 levels are important.

Reply:
We should still flesh this out. (E.g., no reason we can't use "low" or "informational" priorities in a way that avoids an on-call response.) I put a to-do in the doc.